File: tests/Generator/resource/visualbasic/complex/Argument.vb
Imports System.Text.Json.Serialization

' Describes arguments of the operation
Public Class Argument
    <JsonPropertyName("contentType")>
    Public Property ContentType As Nullable(String)

    <JsonPropertyName("in")>
    Public Property _In As Nullable(String)

    <JsonPropertyName("name")>
    Public Property Name As Nullable(String)

    <JsonPropertyName("schema")>
    Public Property Schema As Nullable(PropertyType)

End Class

File: tests/Generator/resource/rust/complex/argument.rs
use serde::{Serialize, Deserialize};
use property_type::PropertyType;

// Describes arguments of the operation
#[derive(Serialize, Deserialize)]
pub struct Argument {
    #[serde(rename = "contentType")]
    content_type: Option<String>,

    #[serde(rename = "in")]
    _in: Option<String>,

    #[serde(rename = "name")]
    name: Option<String>,

    #[serde(rename = "schema")]
    schema: Option<PropertyType>,

}

